### PR TITLE
refactor: remove upper cases in import package names #908

### PR DESCRIPTION
**Pull request by aks3213**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Closes: https://github.com/zeta-chain/node/issues/908

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Reefactor

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Include instructions and any relevant details so others can reproduce. 

- [x] Tested CCTX in localnet
- [ ] Tested in development environment
- [ ] Go unit tests
- [ ] Go integration tests
- [ ] Tested via GitHub Actions 

# Checklist:

- [ ] I have added unit tests that prove my fix feature works

### CODE REVIEW

**Comment by lumtis** [comment]
Great @aks3213 , will have a look

**Review by kingpinXD — APPROVED** [review]
Thanks for the PR 
LGTM

**Comment by billyyang423** [comment]
Can you merge in the latest changes and fix the conflicts, the work I just did fixes the CI problems you ran into because they are coming from a fork

**Comment by aks3213** [comment]
There are many changes in the original branch and it is hard to resolve conflicts manually. Need to discard this PR again and raise it again with updated changes.

**Comment by lumtis** [comment]
> There are many changes in the original branch and it is hard to resolve conflicts manually. Need to discard this PR again and raise it again with updated changes.

I don't think the conflicts are very hard to resolve, I can help with it.

It's very likely that there will always be conflicts to solve since the PR changes many files and the project is currently under active development.

**Comment by aks3213** [comment]
@lumtis I have resolved merge conflicts and have updated my PR. Can you plz check?

File: zetaclient/evm_client.go [hunk 1]
```diff
@@ -26,8 +26,9 @@ import (
 	ethtypes "github.com/ethereum/go-ethereum/core/types"
 
 	erc20custody "github.com/zeta-chain/protocol-contracts/pkg/contracts/evm/erc20custody.sol"
+	zetaconnector "github.com/zeta-chain/protocol-contracts/pkg/contracts/evm/zetaconnector.non-eth.sol"
```
Comment (lumtis) [inline zetaclient/evm_client.go:29]:
```suggestion
```
I think there is a build error because duplicated import

Comment (aks3213) [reply zetaclient/evm_client.go:29]: I have removed duplicate imports.

Comment (aks3213) [reply zetaclient/evm_client.go:29]: @lumtis @kingpinXD please check

**Comment by lumtis** [comment]
Hey @aks3213, thanks again for the PR.
I'm closing this for now due to inactivity. Please feel free to reopen if you're keen on merging.
Given the potential for numerous conflicts with these types of import changes, I think it might make more sense to introduce them progressively across multiple PRs.

**Pull request closed without merging**